File: java/lib/src/test/java/org/xmpt/proto/ProtoTest.java
package org.xmpt.proto;

import org.junit.Test;
import org.xmtp.proto.message.contents.MessageOuterClass.Message;

import static org.junit.Assert.*;

public class ProtoTest {
    @Test public void versionMethodReturnsTrue() {
        Message message = new MessageV2();
        assertTrue(true, message.hasV2());
    }
}

File: dev/kotlin/tests/protoTest.kt
import org.junit.Test
import org.junit.Assert.*

@RunWith(JUnit4::class)
class ProtoTest {
    @Test
    fun testTypesAreAvailable() {
        assertEquals(org.xmtp.proto.message.api.v1.MessageApiOuterClass.SortDirection.SORT_DIRECTION_UNSPECIFIED.rawValue, 0)
    }
}